model/task/results.go

```diff
@@ -133,3 +134,38 @@ func FilterTasksOnStatus(tasks []Task, statuses ...string) []Task {
 
 	return out
 }
+
+type Result struct {
```

Comment:
We should avoid using the word "result" in these struct names to avoid confusing these with test results

Reply:
done.